hack/flake-report-creator.sh

```diff
@@ -6,7 +6,7 @@ docker run -v /etc/pki:/etc/pki -v /etc/ssl:/etc/ssl \
         -e GOOGLE_APPLICATION_CREDENTIALS="$GOOGLE_APPLICATION_CREDENTIALS" \
         -v "${tmp_dir}:/tmp:Z" \
         --network host \
-        quay.io/kubevirtci/flake-report-creator:v20220630-5050e576 \
+        quay.io/kubevirtci/flake-report-creator:v20220706-f742d17f \
```

Comment:
Just to confirm - this image was built with the changes included in this PR?

Reply:
Yes, exactly!